calculator-project/src/components/buttons/buttons.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import "../buttons/buttons.css";
+import { create, all } from "mathjs";
+// import numeral from "numeral";
+
+const config = { };
+const math = create(all, config);
+
+class Buttons extends React.Component{
+    state= {
+        NumberButtons: [{oName: "zero", value: "0"}, {oName: "one", value: "1"}, {oName: "two", value: "2"}, {oName: "three", value: "3"}, {oName: "four", value: "4"}, {oName: "five", value: "5"}, {oName: "six", value: "6"}, {oName: "seven", value: "7"}, {oName: "eight", value: "8"}, {oName: "nine", value: "9"}],
```

Comment:
You didn't have to keep your `NumberButtons` and your `OperatorButtons` in the state, since they don't change. You could just make them variables outside of state and keep your state clean :)